### DISCUSSION TITLE
[A-L] (2023/24) Foglio 12 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#338] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-28 132249](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/5460a64f-f514-4898-b92f-cefda589a280)

**Comment by daromacarlo** [comment]
![Immagine 2023-11-30 174655](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/962fe6f4-625a-45b0-a1b2-b298a2fc3275)
![Immagine 2023-11-30 174716](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/f57bc895-bd87-4004-9797-62333502c3ec)
![Immagine 2023-11-30 174731](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/8db70037-2f53-4a71-aec2-a6d4844e4d6e)
![Immagine 2023-11-30 174748](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/21d448ba-7042-424c-be68-577f2d3cc3e5)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/6d5a96e6-7fdb-4231-aeaa-9d28bc257d2f)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/03638a97-5460-4d40-91e4-6ad46e713912)
